jta-1_2-standalone/pom.xml

```diff
@@ -158,15 +122,18 @@
                     <target>${maven.compiler.target}</target>
                 </configuration>
             </plugin>
-
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${version.maven-surefire-plugin}</version>
                 <configuration>
                     <systemPropertyVariables>
-                        <com.arjuna.ats.arjuna.objectstore.objectStoreDir>${project.build.directory}/defaultstore</com.arjuna.ats.arjuna.objectstore.objectStoreDir>
-                        <com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>${project.build.directory}/commstore</com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
+                        <com.arjuna.ats.arjuna.objectstore.objectStoreDir>
+                            ${project.build.directory}/tx-object-store
+                        </com.arjuna.ats.arjuna.objectstore.objectStoreDir>
+                        <com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
+                            ${project.build.directory}/tx-object-store
+                        </com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
```

Comment:
Ah right. It’s in the other quickstart. I’ll take it out too.
> On 1 Jun 2016, at 12:31, Michael Musgrove <notifications@github.com> wrote:
> 
> In jta-1_2-standalone/pom.xml <https://github.com/jbosstm/quickstart/pull/166#discussion_r65337317>:
> 
> >              <plugin>
> >                  <groupId>org.apache.maven.plugins</groupId>
> >                  <artifactId>maven-surefire-plugin</artifactId>
> >                  <version>${version.maven-surefire-plugin}</version>
> >                  <configuration>
> >                      <systemPropertyVariables>
> > -                        <com.arjuna.ats.arjuna.objectstore.objectStoreDir>${project.build.directory}/defaultstore</com.arjuna.ats.arjuna.objectstore.objectStoreDir>
> > -                        <com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>${project.build.directory}/commstore</com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
> > +                        <com.arjuna.ats.arjuna.objectstore.objectStoreDir>
> > +                            ${project.build.directory}/tx-object-store
> > +                        </com.arjuna.ats.arjuna.objectstore.objectStoreDir>
> > +                        <com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
> > +                            ${project.build.directory}/tx-object-store
> > +                        </com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
> What about this query?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/jbosstm/quickstart/pull/166/files/169bfb6383c480f1b186c64caa0c25c9b36ae3eb#r65337317>, or mute the thread <https://github.com/notifications/unsubscribe/AAL_K-NI1WXLkzm4ZVuCoGodF9ltTdASks5qHV-WgaJpZM4IjPJu>.
>